server/main.py

```diff
@@ -7,6 +7,23 @@
 from fastapi.encoders import jsonable_encoder
 import os
 import shutil
+import psycopg2
+import time
+
+
+def init_sql():
+    with conn.cursor() as cursor:
+        cursor.execute(
+            'CREATE TABLE applications (id INT PRIMARY KEY, name VARCHAR(64))')
```

Comment:
I'd suggest creating database not by server, but with help of db container itself https://levelup.gitconnected.com/creating-and-filling-a-postgres-db-with-docker-compose-e1607f6f882f